src/ansys/mapdl/core/mesh_grpc.py

```diff
@@ -4,12 +4,12 @@
 import weakref
 
 from ansys.api.mapdl.v0 import ansys_kernel_pb2 as anskernel
-from ansys.mapdl.reader.mesh import Mesh
 import numpy as np
 
 from ansys.mapdl.core.common_grpc import DEFAULT_CHUNKSIZE, parse_chunks
 from ansys.mapdl.core.mapdl_grpc import MapdlGrpc
 from ansys.mapdl.core.misc import supress_logging, threaded
+from ansys.mapdl.core.reader import MapdlMesh as Mesh
```

Comment:
@germa89 can you rename the folder from reader to something else (since its a mesh not a reader)

Reply:
That makes sense.